python/pip_install/tools/wheel_installer/wheel_installer.py

```diff
@@ -381,6 +378,33 @@ def _extract_wheel(
         )
         build_file.write(contents)
 
+    with open(
+        os.path.join(installation_dir, "entrypoints.bzl"), "w"
```

Comment:
nit: lets use "entry_points" (two words) throughout instead, just for some marginal more consistency within the codebase. "entrypoint" occurs about 26 times, while "entry_point" occurs about 127.

Reply:
Interesting, I see that `pip` internals are using `entrypoints`, but we are using `entry_points`. Substituted when it comes to our code.